### PR TITLE
Fix error pages not having context-processor data

### PR DESCRIPTION
**Pull request by pmac**
This should fix this issue. The error views included with django-jinja use normal view context instances.

### CODE REVIEW

File: standup/settings.py [hunk 1]
```diff
@@ -124,6 +124,7 @@ def path(*paths):
         },
     },
 ]
+DEFAULT_JINJA2_TEMPLATE_EXTENSION = '.html'
```
Comment (willkg) [inline standup/settings.py]: What does this do?

Comment (pmac) [inline standup/settings.py]: I found that in order to use these convenience error views you need to set this to the extension of your error templates unless you use the default: `.jinja`. See [the code](https://github.com/niwinz/django-jinja/blob/master/django_jinja/views/__init__.py#L51).

Comment (willkg) [inline standup/settings.py]: Fascinating.

**Comment by willkg** [comment]
Sounds good to me!

It probably means we can re-introduce some of the stuff I nixed from the 500 page--namely use of `settings`.

**Comment by pmac** [comment]
I think so too. It's a tough thing to test, but we can make sure it works locally with a view that throws an exception.

**Comment by willkg** [comment]
It's tricky to do that, too. You have to set `DEBUG=False` and then deal with all the things that don't work the same way in that mode.

Way easier to test on -stage.

**Comment by pmac** [comment]
Yeah. Fair point.

**Comment by willkg** [comment]
Sorry, I thought I mentioned this earlier--this looks good to me. r+!

**Pull request merged**